api/src/DataPersister/CategoryDataPersister.php

```diff
@@ -34,4 +36,10 @@ public function beforeCreate($data): BaseEntity {
 
         return $data;
     }
+
+    public function beforeRemove($data): ?BaseEntity {
+        $this->validator->validate($data, ['groups' => ['delete']]);
+
+        return null;
+    }
```

Comment:
Wollen wir das nicht mal in den AbstractDataPersister hochnehmen? Dann könnte ich bei mehr Entities einfach jederzeit beginnen, Delete Validations zu schreiben. Und vielleicht auch gleich für alle Entities einen trivialen DataPersister anlegen, sodass ich das bei allen Entities kann?

Reply:
Mache ich gerne, aber in einem neuen PR.